libmscore/part.cpp

```diff
@@ -598,7 +598,7 @@ void Part::updateHarmonyChannels(bool isDoOnInstrumentChanged, bool checkRemoval
       if (!harmonyChannel() && harmonyCount() > 0) {
             Instrument* instr = instrument();
             Channel* c = new Channel(*instr->channel(0));
-            c->setName(qApp->translate("channel", Channel::HARMONY_NAME));
+            c->setName(Channel::HARMONY_NAME);
```

Comment:
This is what causes translated names to get scored in the score file and subsequently causes the crash at hand in this issue